tests/databases/api_tests.py

```diff
@@ -148,7 +148,6 @@ def test_create_database(self):
         database_data = {
             "database_name": "test-create-database",
             "sqlalchemy_uri": example_db.sqlalchemy_uri_decrypted,
-            "server_cert": ssl_certificate,
```

Comment:
Enabling the `ssl_certificate` fixture raises an error when trying to access the example database on Postgres (as it should), therefore this needs to be removed.